### ISSUE TITLE
117th Congress?

### ISSUE THREAD

**Issue by asebold** [unitedstates/congress-legislators#745]
Was wondering if there's already a branch for updating everything for the 117th Congress. Would be happy to help with it, or get it started. Previously changes for the new Congress were merged to master on Jan 3rd. Is that still the plan for this year?

**Comment by JoshData** [comment]
Hi.

My GovTrack team has started working on it. Normally we're much further along by this point. My goal is to have a draft pull request posted by Friday and then merged on Sunday at noon. We'll get the key data entered but there will probably be a bunch of fields we don't get to. Some fields like official social media accounts (and potentially bioguide IDs) may not even be available until after we merge the PR.

**Comment by asebold** [comment]
Sounds good. Thanks Josh! I'll leave this issue open for a few days in case anyone else is wondering.

**Comment by jlev** [comment]
I see in the PR that legislators-district-offices.yaml has only deletions. Any idea on when the new members office contacts will be available? I think this was originally provided by folks like @tcarobruce and @arrighik, as it's most useful for advocacy organizations making calls or office visits.

**Comment by asebold** [comment]
Just wanted to say thanks to the GovTrack team and @JoshData for getting this done so quickly. Any chance we know when BioGuide IDs will be available for new members?

**Comment by JoshData** [comment]
@asebold: @dwillis committed the bioguide IDs for the new House members and I did the 3 new senators, and we now have the LIS IDs of the new senators as well. So I think all of the core data is now in.

Conceivably this might have been easier if I had waited for the new House and Senate XML member data to go up this afternoon rather than collecting member data ahead of time, but I didn't want to leave anything to chance.
